workspaces/kener/plugins/kener-status/src/components/KenerStatusCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { Card, CardContent, Typography } from '@material-ui/core';
+
+export const KenerStatusCard = () => {
+  const { entity } = useEntity();
+  const annotations = entity?.metadata.annotations || {};
+
+  const uptimeBadgeUrl = annotations['kener/uptimeBadge'];
+  const statusBadgeUrl = annotations['kener/statusBadge'];
```

Comment:
What happens if these are missing?

Reply:
Often when that the case a plugin will use this to make the user experience better: https://backstage.io/storybook/?path=/story/feedback-emptystate--missing-annotation